pkg/getEc2Info.go

```diff
@@ -9,50 +9,96 @@ import (
 	"github.com/aws/aws-sdk-go-v2/service/ec2"
 )
 
-
 type EC2Instance struct {
+	InstanceName      string
+	InstanceID        string
+	InstanceType      string
+	InstancePrivateIP string
+}
 
-	InstanceName 		string
-	InstanceID	 		string
-	InstanceType 		string
-	InstancePrivateIP 	string
-
+func Ec2Client() (ec2.Client, context.Context) {
```

Comment:
We can have one Client function common to all

Reply:
Common as in for particular group, for example EC2 will have instance,ami, EBS, etc